OvieMudi/Banka
# #165483391 migrate existing routes to Postgres

### What does this PR do?
- migrate **create account** route

- migrate **accounts update** route

- migrate **delete account** route

- migrate **credit transactions** route

- migrate **debit transactions** route

### Description of Task to be completed?
Ensure the following endpoints are migrated and working
```
POST  /accounts

PATCH  /account/<account-number>

DELETE  /accounts/<account-number>

POST  /transactions/<account-number>/debit

POST  /transactions/<account-number>/credit
```

### How should this be manually tested?
- All endpoints listed above should be tested via Postman

- On the local machine, run  ` npm run test ` for mocha unit testing

### Any background context you want to provide?
This chore will ensure the all previous routes are migrated to Postgres database.

### What are the relevant pivotal tracker stories?
[#165483391 ](https://www.pivotaltracker.com/story/show/165483391 )
